espnet/nets/pytorch_backend/transformer/attention.py

```diff
@@ -239,7 +239,6 @@ def forward(self, query, key, value, mask, expand_kv=False):
                 else:
                     del key_nonpad_mask
                     q, k, v = self.forward_qkv(query, key, value)
-                    del query, key, value
```

Comment:
These variables should not be deleted. If flash attention fails, we need to fall back to the default attention, which still needs those variables.